### DISCUSSION TITLE
a problem about generating chinese article citation key

### DISCUSSION THREAD

**Discussion by jian850123** [retorquere/zotero-better-bibtex#1876] [Q&A] [answered]
when I use better bibtex to generate chinese article citation key,I find that all the title transform to pinyin (Chinese romanization),no matter how I set the preferences.
I want to know how to get it to show only the first three characters in pinyin(Chinese romanization).

**Comment by duncdrum** [comment, marked as answer]
Hi @jian850123 please see responses to this issue https://github.com/retorquere/zotero-better-bibtex/issues/1678 if you still have problems let us know exactly how you configured your citekey format

**Reply by retorquere** [reply]
BTW if use of jieba warrants its own explanation I'm open to creating a separate page on the site, but I'd most definitely be leaning on the people here for content. I have a vague understanding of wat Jieba does.

**Reply by jian850123** [reply]
> 您好@jian850123请参阅对此问题的响应#1678如果您仍然有问题，请告诉我们您如何配置您的引用格式

Thank you very much，I think this format `[Auth]-[year]-[Title:select=1,1:nopunct:split-ideographs:select=1,3]`have solved my problem.But I have a small problem,chinese article have three words but english article have one word.

**Reply by retorquere** [reply]
If you need help with a specific item in your zotero database, I'd need a support log for that item.

**Reply by jian850123** [reply]
> If you need help with a specific item in your zotero database, I'd need a support log for that item.
9KUCIAPR-apse
Excuse me,Just I send Support Log ID to you?

**Reply by retorquere** [reply]
Yep, that would have been OK, but I need one generated from right-clicking the item (the ID will have `-refs-` in it).

**Reply by jian850123** [reply]
> Yep, that would have been OK, but I need one generated from right-clicking the item (the ID will have `-refs-` in it).

<img width="688" alt="截图_20211720091715" src="https://user-images.githubusercontent.com/37820397/126330819-455cbbaa-8b6f-4e28-9f5f-9c66ae51406e.png">
<img width="680" alt="截图_20211720091704" src="https://user-images.githubusercontent.com/37820397/126330834-af141688-0019-4b02-802b-2d4599960df8.png">
Excuse me,I do not know if I get your meaning right.

**Reply by retorquere** [reply]
No problem. Select those two items, right-click, select "Better BibTeX", then "Send Better BibTeX support log".

**Reply by jian850123** [reply]
> No problem. Select those two items, right-click, select "Better BibTeX", then "Send Better BibTeX support log".

Thanks for your detailed guidence.
FGDF2LSU-refs-apse

**Reply by retorquere** [reply]
Ah, and you'd want different key generation for these two? Based on what criterion?

**Reply by jian850123** [reply]
> Ah, and you'd want different key generation for these two? Based on what criterion?



> Ah, and you'd want different key generation for these two? Based on what criterion?

My thought is that the citation key of Chinese and English article both have the same style,like these two,Francks-2021-A and Lu-2021-Dian.The author and title have the same number of words.I want to know whether I could change citation key to get this thought.

**Reply by retorquere** [reply]
I don't understand your desired outcome. You sometimes want to use the 

> Thank you very much，I think this format `[Auth]-[year]-[Title:select=1,1:nopunct:split-ideographs:select=1,3]` have solved my problem.But I have a small problem,chinese article have three words but english article have one word.

Try `[Auth]-[year]-[Title:nopunct:split-ideographs:select=1,3]`

**Reply by jian850123** [reply]
> 我不明白你期望的结果。你有时想使用
> 
> > 非常感谢，我认为这种格式解决了我的问题。但我有一个小问题，中文文章有三个字，但英文文章有一个词。`[Auth]-[year]-[Title:select=1,1:nopunct:split-ideographs:select=1,3]`
> 
> 尝试`[Auth]-[year]-[Title:nopunct:split-ideographs:select=1,3]`
Thanks very much.
`[Auth]-[year]-[Title:nopunct:split-ideographs:select=1,3]`
It completely meets my requirements.I feel very comfortable.
Best wishes.
